templates/deployment.yaml

```diff
@@ -54,6 +54,10 @@ spec:
               value: {{ $.Values.configs.serverport | quote }}
             - name: FORMS_ENDPOINT
               value: {{ $.Values.configs.formsendpoint }}
+            - name: DATAVIEWS_ENDPOINT
+              value: {{ $.Values.configs.dataviewsendpoint }}
+            - name: MERGED_DATASETS_ENDPOINT
+              value: {{ $.Values.configs.mergeddatasetsendpoint }}
```

Comment:
Can we have some documentation on these? And perhapa also a mention on the `values.yaml` `configs` section as well as support for these configurations?
https://github.com/onaio/helm-dirt-tile-server/blob/daae5c943f7d025bf20380cd9b6894e28cf250f3/values.yaml#L71-L77